wordpress-mobile/WordPress-iOS
# [iOS 13] Fix filter tab bar font

Refs #12211. This PR fixes the issue in iOS 13 where the font in FilterTabBar appeared larger than in iOS 12 and below.

This was due to a change in the behaviour of `traitCollectionDidChange`, which is where we were configuring the tab bar font. Previously it would always be called once, but now it is _only_ called on trait change. This meant the font wasn't being configured on iOS 13. We now do it on initialisation of the tab bar.

![filter-tab-bar](https://user-images.githubusercontent.com/4780/63387924-9eeb9880-c39e-11e9-8825-e7e44e0a8712.png)

**To test:**

* On both iOS 12 and iOS 13, build and run and navigate to Blog Posts
* Verify that the filter bar font size matches on both versions

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt` (won't be visible by users, fix before iOS 13 release)
